src/histolab/tiler.py

```diff
@@ -73,7 +73,9 @@ def locate_tiles(
         extraction_mask: BinaryMask = BiggestTissueBoxMask(),
         scale_factor: int = 32,
         alpha: int = 128,
-        outline: str = "red",
+        outline: Union[str, List] = "red",
```

Comment:
I'm not sure this will work properly. We are using a union of string and list for the sake of the retro-compatibility but this can be a problem:
1. This implies a lot of if statement in the code that in turn requires a bunch of unit/integration tests with some edge cases.
2. the outline when it's a list and the length of the list is greater or less than the number of the tiles we will throw several `IndexError` or wrong color according to the size of the lists. See line `124`.

I wonder if we have to simplify this method or in some way introduce a validator for the outline.

If something is not clear please let me know 😄

Reply:
That's a great point! I added a new generator that zips outline and tiles and throws errors for the edge cases. These edge cases were all added to unit tests.